setup.py

```diff
@@ -27,7 +27,7 @@
 README_contents = open(os.path.join(current_dir, 'README.md'),
                        encoding='utf-8').read()
 doclines = README_contents.split("\n")
-dependencies = ['ftfy >= 4', 'msgpack-python', 'langcodes >= 1.4', 'regex >= 2015']
+dependencies = ['ftfy >= 5', 'msgpack-python', 'langcodes >= 1.4', 'regex == 2017.06.23']
```

Comment:
Okay.  This will delay merging just slightly; I have to add pins to the staging lists.